buildSrc/src/main/kotlin/otel.android-library-conventions.gradle.kts

```diff
@@ -1,52 +0,0 @@
-import org.gradle.api.publish.maven.MavenPublication
-import java.net.URI
-
-plugins {
```

Comment:
I'm totally ok removing this for now, but when we end up having more than one project module we may want to bring it back. In the Splunk project it existed to help several modules not repeat themselves.

Reply:
That's a good point, at first I went with a plugin approach because to me it's a bit cleaner than using scripts and it's also more organized for when there's a lot of code needed for compilation and publishing processes that can be reused across different projects. However, since there are not too many things needed for now, and taking into account the consistency with previous gradle-related code as mentioned above, I changed it back to the script format and reintroduced the common android configs as well.

Probably in the future, if we need to add more compilation tools, or even to support other kinds of projects, not only android lib ones, we might want to consider reorganizing these tools in order to avoid adding too many different responsibilities into a single script... I think we can create one that takes care of configuring android-specific parameters and use the root `build.gradle.kts` script to automatically apply it to all android projects! That way we can still avoid repeating ourselves while keeping the scripts small and specific to a single task.